app/components/profile_picture/image.tsx

```diff
@@ -56,7 +58,7 @@ const Image = ({author, forwardRef, iconSize, size, source, url}: Props) => {
 
         const pictureUrl = buildProfileImageUrlFromUser(serverUrl, author);
```

Comment:
To avoid someone potentially removing `lastPictureUpdateAt` in the future because it's unused in the body of the `useMemo`, you might want to change this to use `buildProfileImageUrl` so that it explicitly uses `lastPictureUpdateAt` in here